### DISCUSSION TITLE
Dynamic Routes

### DISCUSSION THREAD

**Discussion by rserafim** [vercel/next.js#38457] [Help]
Hey!

I have the following folder/file structure:
|── pages
|  |── [slug]
|  |── index.tsx

However, I have some problems when I need to access urls like:

localhost:3000/**_IMG_**/icons/icon-72x72.png

> Server Error
TypeError: Cannot read properties of undefined (reading 'slug')

Because it falls into the getStaticProps function and performs a query on the **_IMG_**. How to get around this?

**Comment by rserafim** [comment]
up

**Reply by valcosmos** [reply]
Hello, according to the information u provided, I guess it may be the **[slug]** file has no suffix, u can try `[slug].tsx`.
